glean-core/ffi/src/lib.rs

```diff
@@ -150,6 +151,24 @@ fn from_raw_string_array_and_string_array(
     }
 }
 
+/// Create a Vec<u32> from a raw C uint64 array.
+///
+/// This will return an empty `Vec` if the input is empty.
+///
+/// ## Safety
+///
+/// * We check the array pointer for validity (non-null).
+fn from_raw_int64_array(values: RawInt64Array, len: i32) -> Vec<i64> {
+    unsafe {
+        if values.is_null() || len == 0 {
+            return vec![];
+        }
+
+        let value_slice = std::slice::from_raw_parts(values, len as usize);
+        value_slice.iter().map(|&v| v as i64).collect()
```

Comment:
Why not the `.to_vec()` I proposed? IMO that's match clearer than cast and collect.

Reply:
Ah, sorry about that, it got lost :)